clients/python/llmengine/data_types.py

```diff
@@ -109,6 +109,12 @@ class GetModelEndpointV1Response(BaseModel):
 
 
 class TaskStatus(str, Enum):
+    """
```

Comment:
We're actually going to remove this from the API, don't think it was supposed to be there.

Reply:
Removing it from both CompletionSyncV1Response and CompletionStreamV1Response ? I can modify the docs assuming that's true. All error codes will be communicated via HTTP response codes?

Reply:
Yeah, this is removed in #57